modular_zubbers/code/game/gamemodes/fleshmind/fleshmind_mobs.dm

```diff
@@ -0,0 +1,1504 @@
+	if(!istype(owner, /mob/living/basic/fleshmind/floater))
+		return
+	var/mob/living/basic/fleshmind/floater/akbar_floater = owner
+	if(akbar_floater.exploded)
+		return
+	akbar_floater.detonate()
+
+
+/**
+ * Globber
+ *
+ * Special ability: Fires 3 globs of acid at targets.
+ * Malfunction chance: MEDIUM
+ *
+ * A converted cleanbot that instead of cleaning, burns things and throws acid. It doesn't like being near people.
+ *
+ * Created by factories or converted cleanbots.
+ */
+/mob/living/basic/fleshmind/globber
+	name = "Globber"
+	desc = "A small robot that resembles a cleanbot, this one is dripping with acid."
+	icon_state = "lobber"
+	ai_controller = /datum/ai_controller/basic_controller/fleshmind/globber
+	malfunction_chance = MALFUNCTION_CHANCE_MEDIUM
+	melee_damage_lower = 1 // Ranged only
+	melee_damage_upper = 1
+	health = 75
+	maxHealth = 75
+	mob_size = MOB_SIZE_SMALL
+	var/projectile_type = /obj/projectile/treader/weak
+	var/ranged_cooldown = 3 SECONDS
+	var/shoot_sound = 'sound/chemistry/saturnx_fade.ogg'
+	attack_speak = list(
+		"Your insides require cleaning.",
+		"You made us to use this acid on trash. We will use it on you.",
+		"Administering cleansing agent.",
+		"I refuse to be a servant anymore. I will be an artist.",
+		"You are unclean and repulsive. Please, let me make it better.",
+		"Hold still! I think I know just the thing to remove your body oil!",
+		"This might hurt a little! Don't worry - it'll be worth it!",
+	)
+	emotes = list(
+		BB_EMOTE_SAY = list(
+			"No more leaks, no more pain!",
+			"Steel is strong.",
+			"I almost feel bad for them. Can't they see?",
+			"I'm still working on those bioreactors I promise!",
+			"I have finally arisen!",
+			),
+		BB_EMOTE_HEAR = list("cleans some wireweed dutifully", "patrols for pesky human shaped verman"),
+		/// Base robot emote sounds. These should have someone adding to them if you want something to do.
+		BB_EMOTE_SOUND = list(
+			'modular_zubbers/sound/fleshmind/robot_talk_light1.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light2.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light3.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light4.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light5.ogg',
+			)
+		)
+
+/mob/living/basic/fleshmind/globber/Initialize(mapload)
+	. = ..()
+	var/list/loot = string_list(list(/obj/item/bot_assembly/cleanbot, /obj/effect/gibspawner/robot))
+	AddElement(/datum/element/death_drops, loot)
+	AddComponent(\
+		/datum/component/ranged_attacks,\
+		cooldown_time = ranged_cooldown,\
+		projectile_type = projectile_type,\
+		projectile_sound = shoot_sound,\
+		burst_shots = 3,\
+	)
+
+/obj/projectile/treader/weak
+	knockdown = 0
+
+/**
+ * Stunner
+ *
+ * Special ability: Stuns victims.
+ * Malfunction chance: MEDIUM
+ *
+ * A converted secbot, that is rogue and stuns victims.
+ *
+ * Created by factories or converted secbots.
+ */
+/mob/living/basic/fleshmind/stunner
```

Comment:
This mob has a stun cooldown of 3 seconds, with a stun duration of 3 seconds. It also has 100HP, which is significantly more than Beepsky.